segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorageConfig.java

```diff
@@ -74,6 +74,11 @@
     public static final Property<Long> MIN_TRUNCATE_RELOCATION_SIZE_BYTES = Property.named("truncate.relocate.size.bytes.min", 64 * 1024 * 1024L);
     public static final Property<Integer> MIN_TRUNCATE_RELOCATION_PERCENT = Property.named("truncate.relocate.percent.min", 80);
 
+    public static final Property<Integer> MAX_LATE_REQUEST_THROTTLE_PERCENT = Property.named("throttle.late.percent.max", 75);
+    public static final Property<Integer> MIN_LATE_REQUEST_THROTTLE_PERCENT = Property.named("throttle.late.percent.min", 25);
+    public static final Property<Integer> MAX_LATE_REQUEST_THROTTLE_DURATION = Property.named("throttle.late.max.ms", 1000);
+    public static final Property<Integer> MIN_LATE_REQUEST_THROTTLE_DURATION = Property.named("throttle.late.min.ms", 100);
```

Comment:
I do not believe a user can be reasonably be expected to know how to set these values. If you really believe these numbers are universal. Commit and hard code them with a comment explaining the reasoning. If you do not, we need to find a different design.

Reply:
I do not agree. These numbers are not universal and I do not think that hardcoding is an acceptable alternative. 
They are system dependent and therefore we should have tunable configuration that Pravega system administrators/integrators can change to suite their needs.
We do want to provide reasonable defaults.